src/options.cpp

```diff
@@ -995,16 +995,25 @@ void options_manager::init()
         build_soundpacks_list(), "basic", COPT_NO_SOUND_HIDE
         ); // populate the options dynamically
 
+    add( "SOUND_ENABLED", "general", translate_marker( "Sound Enabled" ),
+        translate_marker( "If true, music and sound are enabled." ),
+        true, COPT_NO_SOUND_HIDE
+        );
+
     add( "MUSIC_VOLUME", "general", translate_marker( "Music volume" ),
         translate_marker( "Adjust the volume of the music being played in the background." ),
         0, 200, 100, COPT_NO_SOUND_HIDE
         );
 
+    get_option("MUSIC_VOLUME").setPrerequisite("SOUND_ENABLED");
```

Comment:
`options.cpp` is not astyle yet, but still:

Whitespaces are missing in between braces here and below.